devsetup/scripts/gen-ansibleee-ssh-key.sh

```diff
@@ -18,7 +18,11 @@ set -ex
 # expect that the common.sh is in the same dir as the calling script
 SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
 NAMESPACE=${NAMESPACE:-"openstack"}
+METADATA_NAME=${METADATA_NAME:-"ansibleee-ssh-key-secret"}
```

Comment:
It is not used with in the script, May be we can add at L44.

Reply:
ah, right, forgot that one.